spec/unit/azure_server_create_spec.rb

```diff
@@ -259,6 +259,8 @@ def test_params(testxml, chef_config, role_name, host_name)
         @server_instance.run
       end
 
+##########  Uncomment the code below to test automatic certificate generation for ssl communication
+=begin
       it 'create with automatic certificates creation if winrm-transport=ssl' do
```

Comment:
Can we mark this test cases as a `pending` ?

Ex: - https://github.com/chef/knife-azure/blob/nim/disable_cert_generation/spec/unit/azure_server_create_spec.rb#L575